src/lwc/gsChecklistIcon/gsChecklistIcon.js-meta.xml

```diff
@@ -0,0 +1,5 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<LightningComponentBundle xmlns="http://soap.sforce.com/2006/04/metadata">
+    <apiVersion>49.0</apiVersion>
```

Comment:
I noticed that another component in this PR is using api v50. All components/classes in this project should be consistent when it comes to api version. If that means, everything stays at 49 that is fine. Otherwise, this PR and other PR's will need updating to extend api version consistency.